.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+sudo: false
+php:
+  - '7.1'
+  - nightly
+
+before_script:
+  - composer self-update
+  - composer install --prefer-dist --no-interaction
+
+script:
+  - phpunit
```

Comment:
more likely to use `./vendor/bin/phpunit` than you will make sure the travis will run proper version of phpunit